php/src/Data/ConfigurationManager.php

```diff
@@ -172,14 +172,18 @@ public function SetBorgBackupHostLocation(string $location) : void {
 
         $isValidPath = false;
         foreach($allowedPrefixes as $allowedPrefix) {
-            if(str_starts_with($location, $allowedPrefix)) {
+            if(str_starts_with($location, $allowedPrefix) && !str_ends_with($location, '/')) {
+                $isValidPath = true;
+                break;
+            }
+            if ($location === '/var/backups') {
```

Comment:
Subdirectories would also be fine from my point of view but your take I'd say :)

Reply:
we'll only allow /var/backups for now :)